### PR TITLE
Allow plugins to extend objectchange view

### LINKED ISSUE #16353: Allow plugins to extend objectchange.html (changeview) template
### NetBox version

v4.0.3

### Feature type

Change to existing functionality

### Proposed functionality

Allow plugins to use the `PluginTemplateExtension` feature to extend the objectchange.html (changeview) template with additional panels. (/netbox/netbox/templates/extras/objectchange.html)

I've tested this locally and it is as simple as adding `{% load plugins %}` to the template along with the panel placements
eg:
```
<div class="row">
  <div class="col col-md-12">
    {% plugin_full_width_page object %}
  </div>
</div>
```

### Use case

I have an internal plugin that I wish to add some panels to the changelog view that will point out some business-related information about changes.

### Database changes

N/A

### External dependencies

N/A

### PR DESCRIPTION
**Pull request by ITJamie**
### Fixes: #16353

Allows plugins to extend objectchange view with panels

### CODE REVIEW

File: netbox/templates/extras/objectchange.html [hunk 1]
```diff
@@ -144,6 +145,14 @@ <h5 class="card-header">{% trans "Post-Change Data" %}</h5>
         </div>
     </div>
 </div>
+<div class="row mb-3">
+  <div class="col col-md-6">
+    {% plugin_left_page object %}
+  </div>
+  <div class="col col-md-6">
+    {% plugin_right_page object %}
+  /div>
```
Comment (jeremystretch) [inline netbox/templates/extras/objectchange.html:154]: Fix HTML tag.
```suggestion
  </div>
```

File: netbox/templates/extras/objectchange.html [hunk 2]
```diff
@@ -144,6 +145,14 @@ <h5 class="card-header">{% trans "Post-Change Data" %}</h5>
         </div>
     </div>
 </div>
+<div class="row mb-3">
```
Comment (jeremystretch) [inline netbox/templates/extras/objectchange.html:148]: The `<div>` elements with class `row` should not have class `mb-3` applied as it results in extraneous vertical margin between cards. (It should have been removed from the existing elements during the v4.0 UI overhaul.)

**Review by jeremystretch — CHANGES_REQUESTED** [review]

File: netbox/templates/extras/objectchange.html (hunk 2 above)
Comment (ITJamie) [reply netbox/templates/extras/objectchange.html:148]: Would you like me to remove `mb-3` from all the others in this template?

Comment (jeremystretch) [reply netbox/templates/extras/objectchange.html]: Yes, but I'll do it real quick.

**Review by jeremystretch — APPROVED** [review]
Thanks @ITJamie!

**Pull request merged**